src/types/redis_zset.h

```diff
@@ -82,15 +82,39 @@ enum ZSetFlags {
   kZSetXX = 1 << 2,
   kZSetReversed = 1 << 3,
   kZSetRemoved = 1 << 4,
+  kZSetGT = 1 << 5,
+  kZSetLT = 1 << 6,
+  kZSetCH = 1 << 7,
 };
 
+typedef struct ZAddFlags {
```

Comment:
@manchurio You can omit `typedef` here.

Reply:
@manchurio Because it's just a C++ type definition:

```
struct Name {
// members
// methods
};

class OtherName {
// members
// methods
};
```

Reply:
> @torwig Could you explain why? In my opinion, I prefer to use "using" instead of 'typedef', but for consistency with other code. I choose this way.

Just use `class A{};` instead of `typedef class A{} A;`. And you do not need care consistent on `typedef`, we will replace them by `using`.

Reply:
OK, I also prefer class.